conanfile.py

```diff
@@ -93,6 +94,12 @@ def build(self):
         cmake.configure()
         cmake.build()
 
+        # Run tests after build
+        env = VirtualRunEnv(self).environment()
+        env.define("CTEST_OUTPUT_ON_FAILURE", "ON")
+        with env.vars(self).apply():
+            cmake.test()
```

Comment:
With a recent conan (>=2.0.15) this can be simplified with the `cmake.ctest()` interface. Specifically:
`cmake.ctest("--output-on-failure")` instead of all these lines.
https://docs.conan.io/2/reference/tools/cmake/cmake.html#conan.tools.cmake.cmake.CMake.ctest

Reply:
Thanks for the tip! Done.